content/docs/integrations.mdx

```diff
@@ -33,7 +33,7 @@ See the pages in this section for more information on our example data sources,
 
 Any external data source integration requires the following settings:
```

Comment:
Side note: I don't think this is accurate. I think the last four settings are optional ("Headers", "Allow Insecure TLS", "Polling Min/Max Delay", "Client TLS Key").

Reply:
Yes, I was actually looking for "Allow Insecure TLS" in the Console UI and couldn't find it at all. I'll submit an issue to update this page, but it's not a high priority in my opinion at the moment.